File: 디자인패턴/java 디자인 패턴/Template/SimpleDisplayArticle.java
import java.util.ArrayList;

public class SimpleDisplayArticle extends DisplayArticleTemplate {
    public SimpleDisplayArticle(Article article) {
        super(article);
    }

    @Override
    protected void title() {
        System.out.println(article.getTitle());
    }

    @Override
    protected void content() {
        ArrayList<String> content = article.getcontent();
        int CntLines = content.size();
        for (int i=0; i<CntLines; i++) {
            System.out.println(content.get(i));
        }
    }

    @Override
    protected void footer() {
        System.out.println(article.getfooter());
    }
}

File: C#/디자인패턴/Teamplate/SimpleDisplayArticle.cs
public class SimpleDisplayArticle : DisplayArticleTemplate
{
    public SimpleDisplayArticle(Article article) : base(article)
    {
    }

    protected override void title()
    {
        Console.WriteLine(article.getTitle());
    }

    protected override void content()
    {
        foreach (String contentItem in article.getContent())
        {
            Console.WriteLine(contentItem);
        }
    }

    protected override void footer()
    {
        Console.WriteLine(article.getFooter());
    }
}